setup/setup.sh

```diff
@@ -1,9 +1,22 @@
 #!/bin/sh
 
+PRESERVECONFIG=0
+if [ -f /opt/traccar/conf/traccar.xml ]
```

Comment:
Are you sure generic shell supports all the expressions that you are using?

Reply:
I am pretty sure any Linux flavour or shell supports the same ["Evaluate conditional expression" test](http://linuxcommand.org/lc3_man_pages/testh.html).

As test platforms I have Ubuntu which uses [dash](https://linux.die.net/man/1/dash) and Fedora which uses [bash](https://linux.die.net/man/1/bash) as the default shell. Both support this. 

I also tested with [Korn Shell (ksh)](https://linux.die.net/man/1/ksh) which is default on OpenBSD and also supports it.

Reply:
Sounds good.

Reply:
Same for [zsh](http://zsh.sourceforge.net/Doc/Release/Conditional-Expressions.html)